### PR TITLE
Removed empty lfn_close function from FileStore.

### PR DESCRIPTION
**Pull request by lukasredynk**
[FileStore] Removed empty lfn_close, calls to it and few unnecessary labels.

Signed-off-by: Lukas Redynk <lukasz.redynk@intel.com>

### CODE REVIEW

File: PendingReleaseNotes [hunk 1]
```diff
@@ -1,6 +1,8 @@
 v10.0.0
 ======
 
+* [FileStore] removed empty lfn_close function and calls to it.
+
```
Comment (liewegas) [inline PendingReleaseNotes]: Not needed since this doesn't change something that is visible to users/admins.

**Comment by liewegas** [comment]
nice cleanup.  Do you mind rebasing on master? Also, ideally the whitespace changes would be left out, or kept in a separate patch.  Thanks!

File: PendingReleaseNotes (hunk 1 above)
Comment (lukasredynk) [inline PendingReleaseNotes]: Ok, I remove it. I saw email about adding change description along commit so I thought this would be the place.

**Comment by lukasredynk** [comment]
Sure, I rebase it and update the pull request. Sorry about those whitespace changes, I think my IDE did some funky stuff with autoformatter, time to switch back to VIM ;)

**Comment by yuyuyu101** [comment]
ping @lukasredynk and we also need to avoid blank style changing

**Comment by liewegas** [comment]
This needs revision, but I don't think it's worth the effort.

**Pull request closed without merging**